spec/game_spec.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+require './lib/game'
+
+RSpec.describe(Game) do
+  let(:game) { Game.new('Mike', 'John') }
+
+  describe '#check_for_win' do
+    it 'returns true if player has winning moves' do
+      game.player1.moves = '123'
+      expect(game.check_for_win(game.player1)).to eql(true)
+    end
+    it 'returns true if player has winning moves' do
```

Comment:
Leave one empty line around it/specify blocks. This helps to separate the expectations from their conditional logic (contexts for instance).
Checkout this link for more clarification. :point_right: https://github.com/rubocop-hq/rspec-style-guide#empty-lines-around-examples

kindly apply to the rest of the `it` block in your code.